.devcontainer/devcontainer.json

```diff
@@ -22,7 +22,7 @@
   "mounts": [
     {
       "source": "devcontainer-cargo-cache",
```

Comment:
```suggestion
      "source": "devcontainer-cargo-cache-${devcontainerId}",
```

As pointed out in devcontainers/templates#117, this seemed necessary.

Reply:
Ah thanks!